proto/kvrpcpb.proto

```diff
@@ -287,13 +317,27 @@ message RawScanRequest {
     Context context = 1;
     bytes start_key = 2;
     uint32 limit = 3;
+    bool key_only = 4;
 }
 
 message RawScanResponse {
     errorpb.Error region_error = 1;
     repeated KvPair kvs = 2;
 }
 
+message RawBatchScanRequest {
+    Context context = 1;
+    repeated bytes start_keys = 2;
```

Comment:
It confused me, could you add some comments for this message?

Reply:
It's for scanning multiple keys with different prefixes at once. start_key_limit is the limit of pairs for each key prefix, and total_limit is the limit of matched pairs for all prefixes. Normally it is the same as (number of prefixes * start_key_limit) but it could be smaller than that if it's necessary

Reply:
Here is actually what why we wanted these. We are trying to use TiKV as a distributed storage backend and build bigtable data model on top of raw kv interface. And we mapped column family + row key + qualifier + timestamp as raw key. So to query the most recent version  of several qualifiers of the same row key. BatchScan can be used in the scenario to scan these records at once.

Reply:
How about set end_key as region's end_key?

Reply:
can you add comments to explain start_keys and each_limit here?

They are not understood intuitively.

Reply:
Sure, let me add one more field all together at last. Since @zhangjinpeng1987 mentioned there could be duplicated results with this, it's better to have optional end keys as well.